ocaml-lsp-server/src/hover_req.ml

```diff
@@ -12,33 +12,63 @@ let environment_mode =
   | Some true -> Extended_variable
   | Some false | None -> Default
 
-let format_contents ~syntax ~markdown ~typ ~doc =
+let print_dividers sections =
+  match sections with
+  | [] -> ""
```

Comment:
This case is useless, `String.concat ~sep:"some sep" []` should already return `""`